test/framework/util.go

```diff
@@ -189,3 +170,17 @@ func MustParseEnabledDisabled(value string) bool {
 		panic(fmt.Errorf("Unknown value for enabled/disabled: %s", value))
 	}
 }
+
+// GetStringMapFromDatatable parse the given datatable to a key/value string map
+func GetStringMapFromDatatable(dt *messages.PickleStepArgument_PickleTable) (map[string]string, error) {
```

Comment:
This method doesn't seem to be used.

Reply:
Nice catch. I did that before merging with latest changes from Marian. will delete it